dipy/segment/clusteringspeed.pyx

```diff
@@ -277,29 +277,28 @@ cdef class QuickBundlesX(object):
         # No need to free node.father, only the current node.
         free(node)
 
-    cdef void _fetch_level(self, CentroidNode* node):
-        cdef Data2D centroid
-        if node.level == self.level:
-            centroid = <float[:self.features_shape.dims[0],:self.features_shape.dims[1]]> &node.centroid[0][0,0]
-            cluster = ClusterCentroid(np.asarray(centroid).copy())
-            cluster.indices = np.asarray(<int[:node.size]> node.indices).copy()
-            self.clusters.add_cluster(cluster)
-
-    @cython.wraparound(True)
-    def get_clusters(self, int level):
-        self.clusters = ClusterMapCentroid()
-        levels = range(self.nb_levels+1)  # Level 0 is the root
-        self.level = levels[level]
-        self.traverse_postorder(self.root, self._fetch_level)
-        return self.clusters
+    # cdef void _fetch_level(self, CentroidNode* node):
```

Comment:
let's delete those commented lines if everyone is happy with manipulating the TreeClusterMap python object instead of the QuickBundlesX Cython object.